lib/util/package-rules/current-version.ts

```diff
@@ -18,26 +18,29 @@ export class CurrentVersionMatcher extends Matcher {
     if (is.undefined(matchCurrentVersion)) {
       return null;
     }
-
-    if (is.nullOrUndefined(currentValue)) {
-      return false;
-    }
```

Comment:
Before the class refactoring, we handled the case where `currentValue` was undefined but `lockedVersion` exists